packages/package_bumbershoot_3_0_8738/.shed.yml

```diff
@@ -1,11 +0,0 @@
-categories:
-- Tool Dependency Packages
-description: Contains a tool dependency definition that downloads the Bumbershoot tools.
-long_description: |
```

Comment:
Don't forget to increase the tool version number. Need to go to bed, see you tm.